living_documentation_generator/generator.py

```diff
@@ -65,6 +65,7 @@ def __init__(
         github_token: str,
         repositories: list[ConfigRepository],
         project_state_mining_enabled: bool,
+        structured_output: bool,
```

Comment:
How about having as input action_inputs and not persisting the same values in similar variables?

Reply:
Look at the suggested change and please write your opinion. Mixed feelings, but I like it more I guess.

Reply:
I agree with action_inputs related changes.